plutarch-extra/plutarch-extra.cabal

```diff
@@ -65,19 +66,23 @@ common c
     -Wincomplete-record-updates -Wmissing-deriving-strategies
     -Wno-name-shadowing -Wunused-foralls -fprint-explicit-foralls
     -fprint-explicit-kinds -Wno-unrecognised-warning-flags
+    -Wno-missing-export-lists
```

Comment:
Why?

Reply:
This is a weird leftover from earlier Plutarch days. Should be removed I think.